[32mINFO    [0m WebUIAuto:log.py:41  ----- 单据流程开始 ----- 
[31m[1mERROR   [0m WebUIAuto:log.py:47 出现异常，异常信息为：<class 'selenium.common.exceptions.TimeoutException'>
[32mINFO    [0m WebUIAuto:log.py:41 进行截图操作